config.go

```diff
@@ -128,7 +386,58 @@ func loadLndConfig(preCfg *Config) (*lnd.Config, error) {
 		log.Warnf("%v", configFileError)
 	}
 
-	return cleanCfg, nil
+	return cfg, nil
+}
+
+// validateRemoteModeConfig validates the terminal's own configuration
+// parameters that are only used in the "remote" lnd mode.
+func validateRemoteModeConfig(cfg *Config) error {
+	r := cfg.Remote
+
+	// Validate the network as in the remote node it's provided as a string
+	// instead of a series of boolean flags.
+	if _, err := lndclient.Network(r.Lnd.Network).ChainParams(); err != nil {
+		return fmt.Errorf("error validating lnd remote network: %v", err)
+	}
+	cfg.network = r.Lnd.Network
+
+	// If the provided lit directory is not the default, we'll modify the
+	// path to all of the files and directories that will live within it.
+	litDir := lnd.CleanAndExpandPath(cfg.LitDir)
+	if litDir != defaultLitDir {
+		r.LitTLSCertPath = filepath.Join(litDir, defaultTLSCertFilename)
+		r.LitTLSKeyPath = filepath.Join(litDir, defaultTLSKeyFilename)
+		r.LitLogDir = filepath.Join(litDir, defaultLogDirname)
+	}
+
+	r.LitTLSCertPath = lncfg.CleanAndExpandPath(r.LitTLSCertPath)
+	r.LitTLSKeyPath = lncfg.CleanAndExpandPath(r.LitTLSKeyPath)
+	r.LitLogDir = lncfg.CleanAndExpandPath(r.LitLogDir)
+
+	// Make sure the parent directories of our certificate files exist. We
+	// don't need to do the same for the log dir as the log rotator will do
+	// just that.
+	if err := makeDirectories(filepath.Base(r.LitTLSCertPath)); err != nil {
```

Comment:
This should be `filepath.Dir` instead of `filepath.Base`. This is currently effectively running `mkdir tls.cert` and `mkdir tls.key`, which left me with quite the head scratcher when running `litd` from the same directory as my certs 😄

Reply:
Oh, thanks for reporting this :grinning: Fixed it (in the PR, new RC will probably take a few days).

Reply:
Excellent. Many thanks!